op-e2e/system_adminrpc_test.go

```diff
@@ -172,6 +177,54 @@ func TestLoadSequencerStateOnStarted_Started(t *testing.T) {
 	assertPersistedSequencerState(t, stateFile, node.StateStarted)
 }
 
+func TestPostUnsafePayload(t *testing.T) {
+	InitParallel(t)
+	ctx := context.Background()
+
+	cfg := DefaultSystemConfig(t)
+	cfg.Nodes["verifier"].RPC.EnableAdmin = true
+
+	sys, err := cfg.Start(t)
+	require.NoError(t, err)
+	err = sys.BatchSubmitter.Stop(ctx)
+	require.NoError(t, err)
+	defer sys.Close()
+
+	l2Seq := sys.Clients["sequencer"]
+	l2Ver := sys.Clients["verifier"]
+	rollupRPCClient, err := rpc.DialContext(ctx, sys.RollupNodes["verifier"].HTTPEndpoint())
+	require.Nil(t, err)
+	rollupClient := sources.NewRollupClient(client.NewBaseRPCClient(rollupRPCClient))
+
+	time.Sleep(time.Duration(cfg.DeployConfig.L2BlockTime+1) * time.Second)
+	seqBlock, err := l2Seq.BlockByNumber(ctx, nil)
+	require.NoError(t, err)
+	require.Greater(t, seqBlock.Number().Uint64(), uint64(0), "Sequencer should have advanced at least one block")
+	verBlock, err := l2Ver.BlockByNumber(ctx, nil)
+	require.NoError(t, err)
+	require.Equal(t, uint64(0), verBlock.NumberU64(), "Verifier should not have advanced any blocks since p2p & batcher is not enabled")
+
+	blockNumberOne, err := l2Seq.BlockByNumber(ctx, big.NewInt(1))
+	require.NoError(t, err)
+	payload, err := eth.BlockAsPayload(blockNumberOne, sys.RollupConfig.CanyonTime)
+	require.NoError(t, err)
+	err = rollupClient.PostUnsafePayload(ctx, payload)
+	require.NoError(t, err)
+
+	ss, err := rollupClient.SyncStatus(context.Background())
```

Comment:
nit: May as well use the ctx var:
```suggestion
	ss, err := rollupClient.SyncStatus(ctx)
```

Reply:
Thanks for the thorough review, all items fixed now